esmvalcore/config-developer.yml

```diff
@@ -28,7 +28,7 @@ CMIP6:
     ETHZ: '{exp}/{mip}/{short_name}/{dataset}/{ensemble}/{grid}/'
     SYNDA: '{activity}/{institute}/{dataset}/{exp}/{ensemble}/{mip}/{short_name}/{grid}/{latestversion}'
   input_file: '{short_name}_{mip}_{dataset}_{exp}_{ensemble}_{grid}*.nc'
-  output_file: '{project}_{dataset}_{mip}_{exp}_{ensemble}_{short_name}'
+  output_file: '{project}_{dataset}_{mip}_{exp}_{ensemble}_{short_name}_{grid}'
```

Comment:
I think this needs a bit more attention: is `grid` now a mandatory parameter? If it is, we need to add an exception in `_recipe.py` in the form of a `RecipeError` if it's not provided

Reply:
I think for CMIP6 grid has always been mandatory. Do you have a recipe that runs without a specified grid?

Reply:
we can do that later, in a new PR if you think the current behaviour is enough for a new though :+1:

Reply:
You're right we could catch this error before even looking for data, could you open a new issue suggesting that as an improvement?

Reply:
> I changed that recently, look at the code above. You may need to do a git pull if you still get a KeyError.

ah awesome, cheers!

> You're right we could catch this error before

Shall dos! But isn't that a trivial fix by adding `grid` to the `required_keys` list in `_recipe.py` (if we want to promote it to being so important, that is)

Reply:
Feel free to put that information in the issue.

Reply:
Yes, but we could also be a bit smarter and extract `required_keys` from the project DRS